src/e2e-test/java/io/cdap/cdap/ui/stepsdesign/WidgetCSV.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.ui.stepsdesign;
+
+import io.cdap.cdap.ui.types.NodeInfo;
+import io.cdap.cdap.ui.utils.Commands;
+import io.cdap.cdap.ui.utils.Helper;
+import io.cdap.e2e.pages.actions.CdfStudioActions;
+import io.cdap.e2e.utils.ElementHelper;
+import io.cdap.e2e.utils.WaitHelper;
+import io.cucumber.java.en.Then;
+import org.junit.Assert;
+import org.openqa.selenium.By;
+
+public class WidgetCSV {
+//    private NodeInfo projectionNode = new NodeInfo("Projection", "transform", "0");
+    private String sectionName = "drop";
+
+
+
+    @Then("Verify that the CSV Widget contains only one row by default")
+    public void verifyDefaultCSVWidget() {
+        Assert.assertTrue(
```

Comment:
Please reduce the tab or space width - should be 2 like the rest of the code

Reply:
Changed now